### ISSUE TITLE
is there a batch update API 

### ISSUE THREAD

**Issue by tianxiaoliang** [apache/incubator-skywalking#1541]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion

___
### Question
- What do you want to know?
is there a batch update API 

in my case I want to batch update(flush) tracing data instead of call API once for just one single data

**Comment by wu-sheng** [comment]
Batch APIs for what? I assume trace uplinks? 

If so, yes. In gRPC, there is already grpc stream process. In Jetty(http 1.x), the request includes JSONArray.

**Comment by tianxiaoliang** [comment]
yes,  trace uplinks. 
zipkin server doesn't support, so that each tracing data require one http interaction,  that is too much for a service to deal with high concurrency requests

**Comment by tianxiaoliang** [comment]
sorry I found batch update in zipkin server also, thanks

**Comment by wu-sheng** [comment]
What do you mean about `zipkin server`? Are you using Zipkin receiver?

**Comment by tianxiaoliang** [comment]
I was planing to integrate sky walking also, so I am here to ask a question about batch update

**Comment by wu-sheng** [comment]
I recommend you to ask question in Zipkin channel for protocol stuff. They will know better than me. :) I can only provide answers I have known. I am worried to misguide you.

**Comment by tianxiaoliang** [comment]
yes sure, I just wanna ask question, I shouldn't talk about the issue I am facing, that is wierd
